sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -466,19 +451,22 @@ export class ManagementClient extends LinkEntity {
 
   /**
    * Peeks the desired number of messages from the specified sequence number.
-   * @param {Long} fromSequenceNumber The sequence number from where to read the message.
-   * @param {number} messageCount The number of messages to retrieve. Default value `1`.
-   * @param {string} sessionId The sessionId from which messages need to be peeked.
+   *
+   * @param associatedLinkName The name of the associated receiver link
+   * @param fromSequenceNumber The sequence number from where to read the message.
+   * @param messageCount The number of messages to retrieve. Default value `1`.
+   * @param sessionId The sessionId from which messages need to be peeked.
    * @returns Promise<ReceivedMessageInfo[]>
    */
   async peekBySequenceNumber(
+    associatedLinkName: string | undefined,
     fromSequenceNumber: Long,
     maxMessageCount?: number,
     sessionId?: string,
     options?: OperationOptionsBase & SendManagementRequestOptions
   ): Promise<ReceivedMessage[]> {
-    throwErrorIfConnectionClosed(this._context.namespace);
-    const connId = this._context.namespace.connectionId;
+    throwErrorIfConnectionClosed(this._context);
```

Comment:
Do you know if in our tests we have both cases covered, ie: associatedLinkName is set, associatedLinkName is 'undefined'?

Reply:
The client should not care whether associatedLinkName is set or not. It is purely used on the service side for optimization on their end. There is no difference in the client side behavior. So, I don't see a need to test for this case. Thoughts?

Reply:
I think it's fine, I was actually just wondering if this was one of those internal bits that only ever gets used one way in our tests. I don't think we need to change anything for it.